### PR TITLE
edited cifar10_tutorial.py

### LINKED ISSUE #792: Make codeblocks non-executable
Certain codeblocks in the [Training a Classifier colab](https://colab.research.google.com/github/pytorch/tutorials/blob/gh-pages/_downloads/cifar10_tutorial.ipynb#scrollTo=N1BibYLW8C4N) should be non-executable; otherwise, users will run and expect output and be confused when they get an error. Specifically the code under the "2. Define a Convolution Neural Network" and the  "3. Define a Loss function and optimizer" sections. 

They should match the html version: https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py

### PR DESCRIPTION
**Pull request by anikethc**
Fixes #792 

## Description
I've set the code blocks to be non-executable. However, the sections titled "2. Define a Convolution Neural Network" and "3. Define a Loss function and optimizer" should retain their executability to ensure error-free execution. I've also cleaned up the code by removing extraneous symbols , fixed the imaged which was not being rendered correctly and made the links clickable.

### CODE REVIEW

**Comment by anikethc** [comment]
Hi @svekars,
I've made some changes and would appreciate it if you could review them. Thanks!

File: beginner_source/blitz/cifar10_tutorial.py [hunk 1]
```diff
@@ -32,8 +31,7 @@
 ‘dog’, ‘frog’, ‘horse’, ‘ship’, ‘truck’. The images in CIFAR-10 are of
 size 3x32x32, i.e. 3-channel color images of 32x32 pixels in size.
 
-.. figure:: /_static/img/cifar10.png
-   :alt: cifar10
+![](https://pytorch.org/tutorials/_images/cifar10.png)
```
Comment (malfet) [inline beginner_source/blitz/cifar10_tutorial.py:34]: Why this is needed? (I.e. keeping it as is allows one to re-host the repository anywhere they want, isn't it?)

**Review by malfet — CHANGES_REQUESTED** [review]
Please restrict your PR to one fix at the time and perhaps add before and after picture

For example https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html already renders pictures correctly and links are already clickable, so I'm a bit confused why change to the link formatting/ image rendering are necessary

**Comment by anikethc** [comment]
Sorry I will keep that in mind for next issue. I was referring to the colab file the image isnt rendering correctly and all the link are unclickable in the colab https://colab.research.google.com/github/pytorch/tutorials/blob/gh-pages/_downloads/cifar10_tutorial.ipynb#scrollTo=N1BibYLW8C4N .

**Comment by malfet** [comment]
@anikethc at the very least, .py files should remain valid executable files, which is not the case after your changes, is it?

**Comment by anikethc** [comment]
Everything was working fine when I executed the ipynb in the google colab https://colab.research.google.com/drive/1GOhFgmRs-nMF9W7eJjcmBahk6SbBJLXS?usp=sharing maybe something went wrong when I converted it into .py file. I will look into it. Thank you for being patient with me.

**Comment by anikethc** [comment]
Hi @malfet , Could you please review the commit.

**Review by malfet — CHANGES_REQUESTED** [review]
Can you please restrict this PR to just making blogs non-executable?

I.e. the convention throughout the tutorials are to use rst markup convention, not HTML one, which makes lots of `<IMG` and `<A HREF` changes wrong. I understand that they do not render in the collab notebook correctly, but this probably tells something about converter pipeline
